.github/workflows/preliminary_checks.yml

```diff
@@ -70,3 +75,23 @@ jobs:
             echo "\`\`\`" >> $GITHUB_STEP_SUMMARY
             exit 1
           fi
+
+  title-check:
+    name: CC PR Titel
```

Comment:
typo

Reply:
Could also just call "PR Format" for consistency with other jobs names "Code Format", ...